.tool-versions

```diff
@@ -0,0 +1 @@
+java temurin-21.0.3+9.0.LTS
```

Comment:
jeg bruker asdf for å håndtere java versjon. Håper det er greit å legge inn en fil for det 🙏

Reply:
Ligger i ft-inntektsmelding og så bør vel gå fint det